src/components/Web3ReactManager/index.tsx

```diff
@@ -54,7 +54,7 @@ const Web3ReactManager = ({ children }) => {
   // If eager-connect failed, try to connect to network in the URL
   // If no chain in the URL, fallback to default chain
   useEffect(() => {
-    if (triedEager && !networkActive && rpcUrls) {
+    if (!networkActive && rpcUrls) {

```

Comment:
You really want rid of this dont you?

Reply:
ups, doing rebase and using last commit only nowwwww